website/events/templates/events/event.html

```diff
@@ -132,18 +132,14 @@ <h1 class="text-center section-title">{{ event.title }}</h1>
                                         <td>
                                             {% if registration.is_paid %}
                                                 {% if registration.payment.type == payment_method_tpay %}
-                                                    {% trans "You pay with Thalia Pay" %}
+                                                    {% trans "You are paying with Thalia Pay" %}
                                                 {% else %}
                                                     {% trans "You have paid" %}
                                                 {% endif %}
                                             {% else %}
                                                 {% trans "You have not paid yet" %}
                                                 {% if registration.member.tpay_enabled %}
```

Comment:
Maybe the `tpay_enabled` check should be in the template tag

Reply:
I just didn't think about it :p